frontend/src/components/ViewTrips/create-new-trip.js

```diff
@@ -1,16 +1,125 @@
+import * as firebase from 'firebase/app';
+
+import { COLLECTION_TRIPS } from '../../constants/database.js';
+import { getCurUserEmail, getUidFromUserEmail } from './temp-auth-utils.js'
+
+/**
+ * Return a string containing the trip name given the trip name entered in the
+ * add trip form.
+ *
+ * @param {string} rawName String containing trip name entered in add trip form.
+ * @return {string} Cleaned trip name.
+ */
+export function getTripName(rawName) {
+  return rawName === '' ? 'Untitled Trip' : rawName;
+}
+
+/**
+ * Return a string containing the trip destination given the trip destination
+ * entered in the add trip form.
+ *
+ * @param {string} rawName String containing trip description entered in the
+ *     add trip form.
+ * @return {string} Cleaned trip description.
+ */
+export function getTripDestination(rawDestination) {
+  return rawDestination === '' ? 'No Destination' : rawDestination;
+}
```

Comment:
Yeah I could definitely apply this to the two functions you highlighted at least for right now since those are purely text (not special like dates or list of emails (will have own special verification for emails)).

I will add a file to the Utils folder with this.

Reply:
Let me know what you think.